src/doc/gui_manual/Operators/Operators_that_Generate_New_Variables.rst

```diff
@@ -28,16 +28,16 @@ Operators that generate Scalars:
 #.  Connected Components
 #.  DataBinning
 #.  Flux
-#.  IntegralCurve
-#.  LCS
-#.  LimitCycle
+#.  :ref:`Integral Curve operator`
```

Comment:
On the Operators page and Operator Types page, they should be located under Parallel Integral Curve System. It seemed fitting to me to group the four operators together under this headline because they are so similar. However, that means they are not individually exposed in the list of operators. For example, the user has to know to look within the PICS heading to find the Limit Cycle operator. I tried "unravelling" just this heading, but I could not find a way to do that without "unravelling" all the other operators too.

Reply:
I did the grouping because as Eddie described - they are all siblings.